schema/vrs.yaml

```diff
@@ -184,10 +184,37 @@ definitions:
         oneOf:
           - $ref: "#/definitions/MolecularVariation"
           - $ref: "#/definitions/Feature"
-          - $ref: "#/definitions/Location"
+          - $ref: "#/definitions/SequenceExpression"
```

Comment:
Please leave Location in until we agree to remove it.

Without Location, how would we refer to CopyNumber of a region? Via a DerivedSequenceExpression at a Location?

Reply:
I'm seeing this same comment twice–resolving this thread as duplicate.